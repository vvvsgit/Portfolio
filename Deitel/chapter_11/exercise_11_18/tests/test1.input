(800) 555-1212
