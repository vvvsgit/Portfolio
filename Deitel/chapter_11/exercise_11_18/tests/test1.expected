Enter phone number in the form (123) 456-7890:
The phone number entered was: (800) 555-1212
